src/checks/__test__/validateJson.test.ts

```diff
@@ -0,0 +1,160 @@
+import { decodeJson, parseJson, readFileSync } from "../validateJson";
+import { right } from "fp-ts/lib/Either";
+import { VersionInfo } from "../../../generated/definitions/content/VersionInfo";
+import { Zendesk } from "../../../generated/definitions/content/Zendesk";
+
+const expectedValidJson =
+  "{\n" +
+  '  "min_app_version": {\n' +
+  '    "ios": "1.27.0",\n' +
+  '    "android": "1.27.0"\n' +
+  "  },\n" +
+  '  "min_app_version_pagopa": {\n' +
+  '    "ios": "0.0.0",\n' +
+  '    "android": "0.0.0"\n' +
+  "  },\n" +
+  '  "latest_released_app_version" : {\n' +
+  '    "ios": "2.1.0.1",\n' +
+  '    "android": "2.1.0.1"\n' +
+  "  },\n" +
+  '  "rollout_app_version" : {\n' +
+  '    "ios": "0.0.0",\n' +
+  '    "android": "0.0.0"\n' +
+  "  }\n" +
+  "}";
```

Comment:
```suggestion
const expectedValidJson = `{
  "min_app_version": {
    "ios": "1.27.0",
    "android": "1.27.0"
  },
  "min_app_version_pagopa": {
    "ios": "0.0.0",
    "android": "0.0.0"
  },
  "latest_released_app_version" : {
    "ios": "2.1.0.1",
    "android": "2.1.0.1"
  },
  "rollout_app_version" : {
    "ios": "0.0.0",
    "android": "0.0.0"
  }
}`;
```
could it be an improvement?
I tested it

Reply:
Better for sure, changed in 76c14b98c4f81e7285f40e025f74d93d18221088